lib/screens/onboarding.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
```

Comment:
although this isn't really a standard, should we stick to a simple format for imports?

```text
dart

flutter and other packages

project
```

example: 
```dart
import 'dart:io';

import 'package:flutter/material.dart';
import 'package:google_fonts/...';

import 'package:admu_student_app/...';
```

Reply:
got it will make changes in a bit !